lib/main_containers.pm

```diff
@@ -136,7 +136,10 @@ sub load_host_tests_docker {
         loadtest 'containers/validate_btrfs';
     }
     load_volume_tests($run_args);
-    loadtest 'containers/buildx' if (is_tumbleweed || is_microos);
+    if (is_tumbleweed || is_microos) {
+        loadtest 'containers/buildx' if (is_tumbleweed || is_microos);
```

Comment:
```suggestion
        loadtest 'containers/buildx';
```

Reply:
Are we supposed to test buildx in SLE too?

Reply:
No but in line 139 the same condition is already applied.